xonsh/lib/subprocess.xsh

```diff
@@ -0,0 +1,19 @@
+"""Xonsh extension of the standard library subprocess module, using xonsh for
+subprocess calls"""
+from xonsh.imphooks import install_hook
+
+install_hook()
+
+from xonsh.tools import XonshCalledProcessError as CalledProcessError
+from xonsh.lib.os import indir
+
+def run(cmd, cwd=None, check=False):
+    """Stub for ``subprocess.run`` like functionality"""
```

Comment:
This isn't really a stub, it is more of a drop-in replacement (where not all functionality has been implemented)